sympy/solvers/solveset.py

```diff
@@ -307,13 +307,14 @@ def inv(trig):
     return (f, g_ys)
 
 
-def _invert_complex(f, g_ys, symbol):
+def _invert_complex(f, g_ys, symbol,check=False):
```

Comment:
What is the check argument for?

Reply:
Actually, it's for multiple `exp`..(i.e. if more than 1 exp)
Btw it's not fully developed, since if we use more than 2 consecutive exp's( i.e. `exp(exp(exp(..)))` ) it's returning Emptyset which is absurd!

Reply:
I don't understand what the check argument is for so I'm not sure but I don't think it should be there. There should be a better way of achieving whatever it is doing.